sys/kern/kern_cheri_revoke.c

```diff
@@ -1,6 +1,3 @@
-#include <sys/cdefs.h>
```

Comment:
Unrelated: this file is missing a license?

Reply:
Oops, I've ask @nwf about the appropriate license